src/unix/linux_like/android/mod.rs

```diff
@@ -3441,6 +3606,9 @@ safe_f! {
     }
 }
 
+#[cfg_attr(termux, link(name = "android-posix-semaphore"))]
+extern "C" {}
```

Comment:
What is this added for?

Reply:
I think this is outside of the scope of `libc`. We are trying to reduce the number of libraries that get implicitly linked, and a third party library that isn't `libc` doesn't seem to make the cutoff.

Are the `sem_*` functions you mentioned not available via bionic in some cases?